.gitattributes

```diff
@@ -0,0 +1,7 @@
+# prototype-kicad-library-ci
+
+# File: .gitattributes
+# Description: Git attributes for the prototype-kicad-library-ci repository.
+
+# filter KiCad projects
```

Comment:
Please change to `scrub KiCad project files` which describes what the filter does, instead of stating that the attributes entry is for a filter (which can be determined from the contents of the entry itself).
```suggestion
# scrub KiCad project files
```